web/src/reducers/errors.js

```diff
@@ -0,0 +1,67 @@
+import {
+  ADMIN_CREATE_USER_SUCCESS,
+  ADMIN_CREATE_USER_ERROR,
+  ADMIN_EDIT_ACCOUNT_SUCCESS,
+  ADMIN_EDIT_ACCOUNT_ERROR,
+  ADMIN_EDIT_ME_SUCCESS,
+  ADMIN_EDIT_ME_ERROR
+} from '../actions/admin';
+
+import { t } from '../i18n';
+
+const getError = (rawMessage, fallback) => {
+  const [errorFamily, message] = rawMessage.split('.');
+  let error = t(`errors.${errorFamily}.${message}`, { defaultValue: false });
+  if (error === false) {
+    error = t(`errors.${message}`, { defaultValue: false });
+  }
+  return error === false ? fallback : error;
+};
```

Comment:
This function pulls apart the error message from the server, with format `family.message`.  It then tries to load the overridden message.  If that fails, then it goes to the default message.  If there's also not a default, it goes with the fallback.